src/excmds.ts

```diff
@@ -5230,8 +5230,15 @@ export async function keyfeed(mapstr: string) {
 //#background_helper
 browser.runtime.onInstalled.addListener(details => {
     if (details.reason === "install") tutor("newtab")
-    else if ((details as any).temporary !== true && details.reason === "update") updatenative(false)
-    // could add elif "update" and show a changelog. Hide it behind a setting to make it less annoying?
+    else if (details.reason === "update") {
+        if ((details as any).temporary !== true) {
+            updatenative(false)
+        } else {
+            // Temporary extension has been updated in place
+            // Open a new tab where Tridactyl will work for convenience
+            tabopen()
```

Comment:
@Rummskartoffel new behaviour is to open a new tab once the extension has reloaded so that you have somewhere where Tridactyl works. Do you think that's still annoying? Do you have any better ideas?

(I have fixed the stale-extension code issue. I think)

Reply:
New tab's fine with me, I just wanted to make sure I don't lose stuff.